deployment/kustomizations/base/deploy-wren-ai-service.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wren-ai-service-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: wren-ai-service
+  template:
+    metadata:
+      labels:
+        app: wren-ai-service
+    spec:
+      containers:
+        - name: wren-ai-service
+          image: ghcr.io/canner/wren-ai-service:0.3.7
+          env:
+          - name: WREN_AI_SERVICE_PORT
+            valueFrom: 
+              configMapKeyRef:
+                name: wren-config
+                key: WREN_AI_SERVICE_PORT
+          - name: OPENAI_BASE_URL
+            valueFrom: 
+              secretKeyRef:
+                name: wrenai-secrets
+                key: OPENAI_BASE_URL
+          - name: OPENAI_API_KEY
+            valueFrom: 
+              secretKeyRef:
+                name: wrenai-secrets
+                key: OPENAI_API_KEY
+          - name: OPENAI_GENERATION_MODEL
+            valueFrom: 
+              configMapKeyRef:
+                name: wren-config
+                key: OPENAI_GENERATION_MODEL
+          - name: QDRANT_HOST
+            valueFrom:
+              secretKeyRef:
+                name: wrenai-secrets
+                key: QDRANT_HOST
+          # QDRANT_PORT is not implemented yet
+          - name: QDRANT_PORT
+            valueFrom:
+              secretKeyRef:
+                name: wrenai-secrets
+                key: QDRANT_PORT
+          # QDRANT_COLLECTION_NAME is not implemented yet
+          - name: QDRANT_COLLECTION_NAME
+            valueFrom:
+              secretKeyRef:
+                name: wrenai-secrets
+                key: QDRANT_COLLECTION_NAME
+          # QDRANT_DIMENTIONS is not implemented yet
+          - name: QDRANT_DIMENTIONS
+            valueFrom:
+              secretKeyRef:
+                name: wrenai-secrets
+                key: QDRANT_DIMENTIONS
+          # QDRANT_HTTP_MODE is not implemented yet
+          - name: QDRANT_HTTP_MODE
+            valueFrom:
+              secretKeyRef:
+                name: wrenai-secrets
+                key: QDRANT_HTTP_MODE
+          # QDRANT_API_KEY is not implemented yet
+          - name: QDRANT_API_KEY
+            valueFrom:
+              secretKeyRef:
+                name: wrenai-secrets
+                key: QDRANT_API_KEY
```

Comment:
Could we remove unimplemented env at this point? thanks

Reply:
I would've keep it so you would not forget to implement these at some point. Its important in k8s environment

Reply:
@qdrddr
* `QDRANT_COLLECTION_NAME` is not required to be set as env, as the collection name would be set to the hash of the MDL (https://docs.getwren.ai/engine/concept/what_is_mdl). Could be deleted.
* `QDRANT_HOST` makes sense, we could leave it here. @cyyeh & I will further discuss if it's needed.
* `QDRANT_API_KEY` makes sense too. I think you refer to the [security](https://qdrant.tech/documentation/cloud/authentication/).
* `QDRANT_DIMENTIONS`, `QDRANT_HTTP_MODE`: what do these two variables refer to ? Could you show me document about these two ?

Reply:
When you implement a Vector DB, each collection filled with embeddings models will have dimensionality value, number of dimensions. Each embeddings model might produce different amount of dimensions. For example with OpenAI the length of the embedding vector will be 1536 for text-embedding-3-small or 3072 for text-embedding-3-large. Other models will have different dimensionality, for example [mxbai-embed-large](https://ollama.com/library/mxbai-embed-large) has 3072 dimensions. Once Qdrant collection created by one embeddings model, it cannot be used with another embeddings also such collection will have dimensionality set and cannot be changed. So its important to provide a correct dimensionality for a given Qdrant collection.

https://platform.openai.com/docs/guides/embeddings/what-are-embeddings

Reply:
Also Qdrant supports `http://` or `httpS://` connections, if you are connecting to the cloud qdrant service, it would be over `https://` and inside of k8s you may want to use unencrypted `http://`

Reply:
cc @cyyeh